physical/raft/raft_test.go

```diff
@@ -203,6 +204,22 @@ func TestRaft_Backend(t *testing.T) {
 	physical.ExerciseBackend(t, b)
 }
 
+func TestRaft_Backend_LargeValue(t *testing.T) {
```

Comment:
I've noticed that we use some variant of `physical.ExerciseBackend*` in all the CRUD testing of physical storage backends. Being that this PR solely applies to Raft, I wasn't sure if it was necessary to add a new method to the `physical` package to test this one-off case.

Reply:
`ExerciseBackend` is used to make sure that the physical backend can perform all routine CRUD operations. Since your changes involve testing for a particular scenario (i.e. with values exceeding the max size), doing it this way is fine.